src/hpo_service.py

```diff
@@ -1,7 +1,7 @@
 import threading
 import json
 from bayes_optuna import optuna_hpo
-from typing import TypedDict
+#from typing import TypedDict
```

Comment:
Please remove this code, we should not be merging commented code

Reply:
updated